src/SpectatorView.Unity/Assets/SpectatorView.Tests/Microsoft.MixedReality.SpectatorView.Tests.msb4u.csproj.meta

```diff
@@ -0,0 +1,12 @@
+fileFormatVersion: 2
+guid: 7945d97ef1f124540a56f8cf48a62b75
+ScriptedImporter:
+  fileIDToRecycleName:
+    11400000: Microsoft.MixedReality.SpectatorView.Tests.msb4u
+  externalObjects: {}
+  userData: 
+  assetBundleName: 
+  assetBundleVariant: 
+  script: {fileID: 11500000, guid: 0e4dca1acba66fd478b3854f09a32ec7, type: 3}
+  buildEngine: 0
+  profiles: []
```

Comment:
Seems like these might be the same .meta files that probably should be excluded by .gitignore?

Reply:
I'll ask andrei about this and will remove them as needed in a follow up review.